### PR TITLE
manjaro.org download cdn logo fix

### PR DESCRIPTION
**Pull request by FadeMind**
https://manjaro.org/download/

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -12078,6 +12078,9 @@ INVERT
 
 manjaro.org
 
+INVERT
+img[id="cdn"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:12082]:
```suggestion
img#cdn
```

**Comment by Gusted** [comment]
Oops this notification slipped into spam/trash. Feel free to ping me after a few days if I had activity on your other PRs.

**Review by Gusted — COMMENTED** [review]
Otherwise looks fine.

**Comment by FadeMind** [comment]
@Gusted  ready to go

**Review by Gusted — APPROVED** [review]

**Pull request merged**